### PR TITLE
Support GCS upload

### PR DESCRIPTION
**Pull request by yongchand**
- Support upload of parquet files to GCS

### CODE REVIEW

File: klaytn-tx-latency-measurement/.env.template [hunk 1]
```diff
@@ -1,5 +1,6 @@
 PRIVATE_KEY=
 CAVER_URL=https://public-node-api.klaytnapi.com/v1/baobab
+UPLOAD_METHOD=AWS
```
Comment (Yeonju-Kim) [inline klaytn-tx-latency-measurement/.env.template:3]: Are GCP_PROJECT_ID and GCP_KEY_FILE_PATH going to be added here?

File: klaytn-tx-latency-measurement/sendtx_klaytn.js [hunk 2]
```diff
@@ -171,9 +211,9 @@ async function sendTx() {
         // console.log(`${data.executedAt},${data.chainId},${data.txhash},${data.startTime},${data.endTime},${data.latency},${data.txFee},${data.txFeeInUSD},${data.resourceUsedOfLatestBlock},${data.numOfTxInLatestBlock},${data.pingTime},${data.error}`)
     }
     try{
-        await uploadToS3(data)
+        await uploadChoice(data)
     } catch(err){
-        console.log('failed to s3.upload! Printing instead!', err.toString())
+        console.log('failed to gcs.upload! Printing instead!', err.toString())
```
Comment (Yeonju-Kim) [inline klaytn-tx-latency-measurement/sendtx_klaytn.js]: If you want to use both uploading methods, this log can be changed like `${process.env.UPLOAD_METHOD === 'AWS'?'s3':'gcs'}.upload!`

File: klaytn-tx-latency-measurement/.env.template (hunk 1 above)
Comment (yongchand) [reply klaytn-tx-latency-measurement/.env.template:3]: Gee, thanks

File: klaytn-tx-latency-measurement/sendtx_klaytn.js (hunk 2 above)
Comment (yongchand) [reply klaytn-tx-latency-measurement/sendtx_klaytn.js]: Just committed! Please let me know if this is what you meant

**Review by Yeonju-Kim — APPROVED** [review]

**Review by Yeonju-Kim — APPROVED** [review]

**Pull request merged**